autoextract_poet/inputs.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from typing import Any, Dict, Optional
+
+
```

Comment:
Are we moving those classes to the web-poet repository?

Reply:
I think yes, that's the way to go. Probably makes sense to create web_poet.inputs.autoextract namespace/package, or maybe web_poet.autoextract.

Reply:
Wouldn't it create some sort of coupling of webpoet and our technology (AutoExtract)?
Maybe a new package?

Reply:
We'll now have  `autoextract-poet` repository for those classes.